ts/Dashboards/Plugins/HighchartsComponent.ts

```diff
@@ -618,11 +619,11 @@ class HighchartsComponent extends Component {
                             .getColumnVisibility(name) !== false :
                         true;
 
-                    if (!isVisible && !columnAssignment[name]) {
-                        return false;
+                    if (!defined(this.options.columnAssignment)) {
+                        return true;
                     }
 
-                    if (columnAssignment[name] === null) {
+                    if (!isVisible || !columnAssignment[name]) {
```

Comment:
This will have issues with empty strings.

Reply:
I will create a ticket.

Reply:
I tested this scenario and all should work properly. If you have data or scenario when not, please let me know.